editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -128,6 +128,7 @@ const VerticalFeatureTabs: React.FC = () => {
             sx={{ width: "100%" }}
             aria-labelledby={`vertical-tab-${i}`}
             id={`vertical-tabpanel-${i}`}
+            data-testid={`vertical-tabpanel-${i}`}
```

Comment:
This felt more explicit for any future changes to this component as it is directly referencing it's use for testing. If this is OTT lemme know and I can switch to the `id` or `aria-labelledby`, but tried to account for the component still being worked on.